katas/content/random_numbers/index.md

```diff
@@ -16,14 +16,14 @@ The quantum algorithm for random number generation is one of the simplest applic
 
 **What you should know to start working on this kata:**
 
-- The concept of qubit and measurement
-- Single-qubit gates
+- Basic knowledge of qubits and measurements. If you need a refresher, you can check the Qubit kata.
+- Basic knowledge of single-qubit gates. If you need a refresher, you can check the Single-Qubit Gates kata.
```

Comment:
Are we updating the format of how we list prerequisites to include pointers to other refresher katas? I'm reluctant to do this, since this will lead to really long lists of prerequisites, especially in the later katas that assume pretty much all single-qubit and multi-qubit katas as a given.

In this particular case, measurements needed for this kata are covered in "Measurement in Single-Qubit Systems" kata, so even for this very basic kata there are three separate prerequisites.

Reply:
I see, each kata is a consequence of the previous katas. Would you be ok with a general sentence at the end of prerequisites like " if you need a refresher of these topics, check out the previous katas"?

Reply:
Yes. This seems intuitive when going through a sequence of tutorials, but one general sentence is better than long lists of pointers to the same katas